Robust.Client/GameObjects/Components/Containers/ContainerManagerComponent.cs

```
@@ -144,7 +144,10 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 // Add new entities.
                 foreach (var uid in data.ContainedEntities)
                 {
-                    var entity = Owner.EntityManager.GetEntity(uid);
+                    if (!Owner.EntityManager.TryGetEntity(uid, out var entity))
```

This really should never be possible. I would rather a test case for this get made and the root bug fixed than ignoring the error here.